certbot/interfaces.py

```diff
@@ -201,7 +201,9 @@ class IConfig(zope.interface.Interface):
     """
     server = zope.interface.Attribute("ACME Directory Resource URI.")
     email = zope.interface.Attribute(
-        "Email used for registration and recovery contact. (default: Ask)")
+        "Email used for registration and recovery contact. (default: Ask). "
+        "Use comma to register multiple emails, "
+        "ex: u1@example.com,u2@example.com")
```

Comment:
In `argparse`'s help output, it adds "(default: <value>)" to the end of parameter to show it's default value. To consistently keep this convention, we should add the "Use comma..." text before (default: Ask).

Reply:
ah I see. Thanks for pointing that out. I will make an update.